flask_security/utils.py

```diff
@@ -158,12 +159,29 @@ def encrypt_password(password):
 
     It uses the configured encryption options.
 
+    .. deprecated:: 2.0.2
+       Use :func:`hash_password` instead.
+
     :param password: The plaintext password to encrypt
     """
+    warnings.warn(
+        'Please use hash_password instead of encrypt_password.',
+        DeprecationWarning
+    )
+    return hash_password(password)
+
+
+def hash_password(password):
+    """Hash the specified plaintext password.
+
+    It uses the configured hashing options.
+
+    :param password: The plaintext password to hash
```

Comment:
Can we also include `.. versionadded` similar to https://github.com/inveniosoftware/flask-security-fork/pull/28/files#diff-87f9321ea75b63ed0a7f694abea714efR162 ?